cmake/Modules/CMakeCargo.cmake

```diff
@@ -22,31 +38,37 @@ function(cargo_build)
         set(LIB_BUILD_TYPE "release")
     else()
         set(LIB_BUILD_TYPE "debug")
-    endif()    
-    
+    endif()
+
     if(WIN32)
-        set(LIB_FILE "target/${LIB_TARGET}/${LIB_BUILD_TYPE}/${LIB_NAME}.lib")
+        set(LIB_FILE "${CARGO_TARGET_DIR}/${LIB_TARGET}/${LIB_BUILD_TYPE}/${LIB_NAME}.lib")
```

Comment:
CMAKE_STATIC_LIBRARY_SUFFIX can be used instead of hardcoding ".lib" and ".a", but we'd need to ensure it doesn't end up being "dylib" on macOS